evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -149,7 +149,7 @@ open class PackageRule(
 
         override val description = "\tEvaluating license rule '$name' for $licenseSource license '$license'."
 
-        override fun issueSource() = "$name - $license ($licenseSource)"
+        override fun issueSource() = "$name - ${pkg.id.toCoordinates()} - $license ($licenseSource)"
```

Comment:
I'd prefer adding a separate attribute for `affectedPackages`.

...Then it is up to the view to decide what is shown.
Mixing data in the model eliminates that flexibility in the view and also in all other consumers.

Reply:
I agree that it makes sense to think about making a more specific class for rule violations with fields like `affectedPackages` (and probably more like `affectedLicense`, ...), but that is a bigger change I cannot do now.

Reply:
ok, assuming this is not used by any matching of resolutions I'm ok with doing this outside of this PR.